desc/backend.py

```diff
@@ -530,7 +588,7 @@ def while_loop(cond_fun, body_fun, init_val):
             val = body_fun(val)
         return val
 
-    def vmap(fun, out_axes=0):
+    def vmap(fun, in_axes=0, out_axes=0):
```

Comment:
When JAX is installed `desc.backend.vmap` is an alias for `jax.vmap`. Otherwise it falls back to this function, which is just a for loop that reproduces the final output of jax.vmap (but it's just a for loop). We should use `desc.backend.vmap` instead of jax.vmap so that DESC still works without JAX installed (in case JAX is abandoned for some new autodiff library I suppose.)

Reply:
Ah, that is good thinking. I gotta change my code because, I use from desc.backend import jnp which will fail if we switch to something other than jax.

Reply:
Using `from desc.backend import jnp` is the correct way to do this. Using `from jax.numpy import jnp` would be wrong.

Reply:
well technically the idea I think is if you need vmap specifically I think you want to do `from desc.backend import vmap` as if we dont have jax installed, `jnp` just becomes an alias for `np` but numpy does not have `vmap`

though I think we can't really run without JAX anyways rn except maybe for just plotting or something